src/lib/routes/client-api/metrics.test.ts

```diff
@@ -209,3 +208,48 @@ test('should set lastSeen on toggle', async () => {
 
     expect(toggle.lastSeenAt).toBeTruthy();
 });
+
+test('should return a 400', async () => {
```

Comment:
Would expand this test description a bit, like `should return 400 when required property X is missing`.